src/MathDistOps/pexp.py

```diff
@@ -0,0 +1,19 @@
+def pexp(x, rate, output_image=False):
+    """
+    Calculates the exponential distribution cumulative probability for a given value.
+
+    Parameters:
+    - x (float): The quantile at which to calculate the cumulative probability.
+    - rate (float): The rate parameter (lambda) of the exponential distribution.
+    - output_image (bool, optional): If True, generates and returns a plot of the exponential 
+      distribution with the cumulative probability highlighted for the given quantile. Default is False.
+
+    Returns:
+    - float: Cumulative probability for the given quantile.
+    - matplotlib.figure.Figure (optional): A plot of the exponential distribution if output_image is True.
```

Comment:
I would rather go with Altair than with mathplotlib. What do u think?
```suggestion
    - altair.Figure (optional): A plot of the exponential distribution if output_image is True.
```

Reply:
Because I remember we add the package matplotlib in your yml file and I think Katherine has used the same packages. But I am ok with either way